tests/ci/cdk/cdk/aws_lc_github_fuzz_ci_stack.py

```diff
@@ -0,0 +1,134 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from aws_cdk import core, aws_codebuild as codebuild, aws_iam as iam, aws_ec2 as ec2, aws_efs as efs
+
+from util.ecr_util import ecr_arn
+from util.iam_policies import code_build_batch_policy_in_json, \
+    code_build_fuzz_policy_in_json
+from util.metadata import AWS_ACCOUNT, AWS_REGION, GITHUB_REPO_OWNER, GITHUB_REPO_NAME
+from util.yml_loader import YmlLoader
+
+
+class AwsLcGitHubFuzzCIStack(core.Stack):
+    """Define a stack used to batch execute AWS-LC tests in GitHub."""
+
+    def __init__(self,
+                 scope: core.Construct,
+                 id: str,
+                 x86_ecr_repo_name: str,
+                 arm_ecr_repo_name: str,
+                 spec_file_path: str,
+                 **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        # Define CodeBuild resource.
+        git_hub_source = codebuild.Source.git_hub(
+            owner=GITHUB_REPO_OWNER,
+            repo=GITHUB_REPO_NAME,
+            webhook=True,
+            webhook_filters=[
+                codebuild.FilterGroup.in_event_of(
+                    codebuild.EventAction.PULL_REQUEST_CREATED,
+                    codebuild.EventAction.PULL_REQUEST_UPDATED,
+                    codebuild.EventAction.PULL_REQUEST_REOPENED)
+            ],
+            clone_depth=1)
+
+        # Define a IAM role for this stack.
+        code_build_batch_policy = iam.PolicyDocument.from_json(
+            code_build_batch_policy_in_json([id])
+        )
+        fuzz_policy = iam.PolicyDocument.from_json(code_build_fuzz_policy_in_json())
+        inline_policies = {"code_build_batch_policy": code_build_batch_policy,
+                           "fuzz_policy": fuzz_policy}
+        role = iam.Role(scope=self,
+                        id="{}-role".format(id),
+                        assumed_by=iam.ServicePrincipal("codebuild.amazonaws.com"),
+                        inline_policies=inline_policies)
+
+        # Create the VPC for EFS and CodeBuild
+        public_subnet = ec2.SubnetConfiguration(name="PublicFuzzingSubnet", subnet_type=ec2.SubnetType.PUBLIC)
+        private_subnet = ec2.SubnetConfiguration(name="PrivateFuzzingSubnet", subnet_type=ec2.SubnetType.PRIVATE)
+
+        # Create a VPC with a single public and private subnet in a single AZ. This is to avoid the elastic IP limit
+        # being used up by a bunch of idle NAT gateways
+        fuzz_vpc = ec2.Vpc(
+            scope=self,
+            id="FuzzingVPC",
```

Comment:
Updated, I left the CodeBuild project as just FuzzingCodeBuild to match the other ones. Is this what shows up in GitHub CI status?

Reply:
I think it's the project name will show up in GitHub CI status.  
This can be confirmed by 
Click `Settings` of awslc repo -> `Branches` -> click `main` branch -> Scroll down to `Status checks found in the last week for this repository`.